# Request control signal
bool request

---
# response is true if value accepted, otherwise server ignore the request and response false
bool response

Chassis value